src/components/JdsSimpleTable/index.js

```diff
@@ -0,0 +1,4 @@
+import JdsSimpleTable from './SimpleTable.vue'
+
+export { JdsSimpleTable }
+export default JdsSimpleTable
```

Comment:
**Low Priority**
It's better to be given a new line

Reply:
thankyou, will fix it asap

Reply:
Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)